includes/Modules/AdSense/AMP_Tag.php

```diff
@@ -122,23 +122,15 @@ public function set_story_ad_slot_id( $ad_slot_id ) {
 	 * @since n.e.x.t
 	 */
 	private function render_story_auto_ads() {
-		printf(
-			'
-				<amp-story-auto-ads>
-					<script type="application/json">
-						{
-							"ad-attributes":
-							{
-								"type": "adsense",
-								"data-ad-client": "%s",
-								"data-ad-slot": "%s"
-							}
-						}
-					</script>
-				</amp-story-auto-ads>
-			',
-			esc_js( $this->tag_id ),
-			esc_js( $this->story_ad_slot_id )
+		$config = wp_json_encode(
+			array(
+				'ad-attributes' => array(
+					'type'           => 'adsense',
+					'data-ad-client' => $this->tag_id,
+					'data-ad-slot'   => $this->story_ad_slot_id,
+				),
+			)
 		);
+		printf( '<amp-story-auto-ads><script type="application/json">%s</script></amp-story-auto-ads>', esc_js( $config ) );
```

Comment:
We shouldn't use `esc_js` here, as that function is only intended for escaping JS within an HTML attribute (it is basically `esc_attr`, specific to JS).

We can instead simply `wp_json_encode` the configuration array here.